playbooks/roles/common-server-init/tasks/networking.yml

```diff
@@ -1,13 +1,15 @@
 ---
 
-- name: Add network interface for floating IP
-  when: COMMON_FLOATING_INTERFACE_NAME and COMMON_FLOATING_IP
-  block:
+- name: Add network interface for floating IP (Xenial, Bionic)
+  when: COMMON_FLOATING_INTERFACE_NAME and COMMON_FLOATING_IP and ansible_distribution_release in ["xenial", "bionic"]
```

Comment:
@giovannicimolin, do we have any dedicated servers running Bionic? If not, we should drop it from the check.

Reply:
@lgp171188 I think we have one or two servers that were upgraded to Bionic due to dependency issues. I'll keep it around just in case. 

When the upgrades are done, we should clear this entire block and remove the check for the Focal branch.